### PR TITLE
Fix ios editor height display incorrectly

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Related issue
- https://github.com/linagora/tmail-flutter/issues/3101#issuecomment-2306298330

### CODE REVIEW

File: lib/src/editor.dart [hunk 1]
```diff
@@ -959,19 +958,6 @@ pre {
...
-    log('HtmlEditorState::_onContentSizeChanged:contentHeight: $contentHeight');
-    final documentHeight = _documentHeight ?? 0;
-    if (contentHeight > documentHeight && mounted) {
-      setState(() {
-        _documentHeight = contentHeight + _offsetHeight;
-      });
-    }
-  }
+    Size newContentSize,
+  ) => onDocumentChanged();
```
Comment (dab246) [inline lib/src/editor.dart:962]: It may affect composer content editing. Please make sure to check again.

Comment (tddang-linagora) [reply lib/src/editor.dart:962]: No issue found for composer

Comment (dab246) [reply lib/src/editor.dart:962]: Please test the `lazy load image` case with `Forward/Reply` emails. Because we have fixed that bug before.

Comment (tddang-linagora) [reply lib/src/editor.dart:962]: Tested and confirmed both `lazy load image` and `rich text`

Comment (dab246) [reply lib/src/editor.dart:962]: Please attach demo on Tmail project side

**Review by dab246 — APPROVED** [review]

File: lib/src/editor.dart (hunk 1 above)
Comment (tddang-linagora) [reply lib/src/editor.dart:962]: Done

**Review by hoangdat — APPROVED** [review]

**Pull request merged**